src/lib/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestPatronLoan/BorrowingRequestLoanExtension/index.js

```diff
@@ -0,0 +1,25 @@
+import { connect } from 'react-redux';
+
+import {
+  borrowingRequestLoanExtensionAccept,
+  borrowingRequestLoanExtensionDecline,
+} from './actions';
+import BorrowingRequestLoanExtensionComponent from './BorrowingRequestLoanExtension';
+
+const mapStateToProps = state => ({
+  isLoading: state.borrowingRequestLoanExtension.isLoading,
+  error: state.borrowingRequestLoanExtension.error,
+  hasError: state.borrowingRequestLoanExtension.hasError,
+});
+
+const mapDispatchToProps = dispatch => ({
+  borrowingRequestLoanExtensionAccept: (brwReqPid, loanEndDate) =>
```

Comment:
If it makes sense again, instead of have different methods `borrowingRequestLoanExtensionAccept` and `borrowingRequestLoanExtensionDecline` could we use the same `borrowingRequestLoanExtension` with an extra param action="accept" or "decline"?

again if it is trivial

Reply:
I also prefer methods to strings: you get build errors and not runtime errors.